cli/crates/cli/tests/pagination.rs

```diff
@@ -4,301 +4,171 @@ mod utils;
 
 use serde_json::{json, Value};
 use utils::consts::{
-    PAGINATION_MUTATION, PAGINATION_PAGINATE_TODOS, PAGINATION_PAGINATE_TODO_LISTS, PAGINATION_SCHEMA,
+    PAGINATION_CREATE_TODO, PAGINATION_CREATE_TODO_LIST, PAGINATION_PAGINATE_TODOS, PAGINATION_PAGINATE_TODO_LISTS,
+    PAGINATION_SCHEMA,
 };
 use utils::environment::Environment;
 
 #[test]
-
-fn pagination() {
+fn test_pagination() {
```

Comment:
Why was the name here changed? This makes the output a bit less clean since it shows up in certain locations (and you're running tests to begin with)

Reply:
Ah force of habit, changing it